### ISSUE TITLE
KeyError: 5015

### ISSUE THREAD

**Issue by mkody** [frozenpandaman/splatnet2statink#1]
I just got the key error after a battle. From the Ninty app and by looking at my json, the weapon with id 5015 is... Hero Dualie Replicas (`heromaneuver_replica` [from stat.ink](https://stat.ink/api-info/weapon2?_lang_=en-US)), which looks already listed with id 5040.
So, what is 5040?

```
Traceback (most recent call last):
  File "splatnet2statink.py", line 271, in <module>
    ally_stats.append(translate_weapons[int(battledata["my_team_members"][n]["player"]["weapon"]["id"])])
KeyError: 5015
```

Here's the value of `battledata`: https://gist.githubusercontent.com/mkody/d27602076b430e40ada6f34c8bfea154/raw/366343c2e179a481b5f1574a7f91441ed6c17f0f/lint-battledata.json

**Comment by clovervidia** [comment]
I was wondering that myself when I saw that all the other replicas had an ID that was 5 + the ID of their base weapon. Someone probably fat-fingered something because I don't think 5040 is a weapon... yet. Good catch.

**Comment by frozenpandaman** [comment]
Sorry for the trouble, @mkody! That someone was probably me...

Seems @clovervidia has fixed this in 71150bf so I'll close this for now.

**Comment by frozenpandaman** [comment]
^ Haha, whoops...
